stable/node-feature-discovery/templates/daemonset.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "node-feature-discovery.fullname" . }}
+  labels:
+{{ include "node-feature-discovery.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+{{ include "node-feature-discovery.matchLabels" . | indent 6 }}
```

Comment:
This will run only on nodes with `app.kunernetes.io/name=<chart name>`
How would that work? Are you counting on something to know that chart name and label the modes beforehand? If so, why? I would expect this to be empty by default

Reply:
> This will run only on nodes with app.kunernetes.io/name=<chart name>

probably you confused with `nodeSelector`

Reply:
I remember that logic is same as for `selector` in `Service`.

The simplest explanation which I found `Label selector that determines which Pods belong to the DaemonSet`.

Reply:
Ahh confused indeed. My bad, ignore